vizro-core/examples/demo/from_yaml/dashboard.yaml

```diff
@@ -48,7 +54,7 @@ pages:
       - figure:
           _target_: choropleth
           color: lifeExp
-          data_frame: gapminder
+          data_frame: df
```

Comment:
I actually think `gapminder` here is much nicer and more descriptive than `df` so I would change this back unless you had a very good reason for it.

Reply:
Mostly to keep aligned with the naming in the pydantic version, but I can change it there as well